src/template/minimal/_markbind/layouts/default/footer.md

```diff
@@ -0,0 +1 @@
+<footer/>
```

Comment:
Could we use
```
<footer/header/navigation>
---empty line---
</footer/header/navigation>
```

in the minimal template files instead?
So if the user wants to add content it'll be slightly simpler

Reply:
@ang-zeyu Yeah, that will do.